cpp/src/holtwinters/internal/hw_eval.cuh

```diff
@@ -15,9 +15,13 @@
  */
 
 #pragma once
-#include <raft/cudart_utils.h>
+
+#include "cuml/common/device_buffer.hpp"
```

Comment:
We should probably deprecate this header anyway. Rather than including it, would you mind just updating the type from `MLCommon::device_buffer` to `raft::mr::device::buffer`?

Reply:
clangd auto-inserted this header (hence the quotes which should be brackets).  You really should use `rmm::device_buffer`.